### ISSUE TITLE
sanitizeInput not working for textarea

### ISSUE THREAD

**Issue by alexmiddeleer** [DockYard/ember-one-way-controls#112]
Hi there, 

I tried using this addon to create a textarea that I can sanitize on the fly, but while my sanitize function fires and returns a new value, it is unable to modify the visible content of the textarea on the fly. The value attribute and the parent component's property that it's bound to are being modified correctly, but the text the user sees doesn't change.

I think this may be impossible thanks to browser limitations around textarea elements, but I figured if anyone had come across it, perhaps you had.

```
  {{one-way-textarea
      contactRequest.explanation_text
      rows=4
      update=(action (mut contactRequest.explanation_text))
      sanitizeInput=(action "limitInput")
      placeholder=(t 'contactRequestDialog.explanation')
      standardizedMaxLength=maxExplanationLength
      }}

...

    limitInput(value) {
      let maxLen = 10;
      if ((typeof maxLen === 'number') && value && value.length && (value.length > maxLen)) {
        return value.substr(0, maxLen);
      }
      return value;
    }
```

**Comment by martndemus** [comment]
Yeah, I'm going to deprecate sanitize input, you should sanitize the input on the update callback instead. See my suggestion on #103.

**Comment by alexmiddeleer** [comment]
Thanks for the response!

**Comment by martndemus** [comment]
I'm going to come back on my previous statement:
This should now be fixed and just working in `1.0.2`
